cmd/server-startup-msg.go

```diff
@@ -79,39 +80,41 @@ func printStartupMessage(apiEndpoints []string, err error) {
 	}
 }
 
-// Returns true if input is not IPv4, false if it is.
-func isNotIPv4(host string) bool {
+// Returns true if input is IPv6
+func isIPv6(host string) bool {
 	h, _, err := net.SplitHostPort(host)
 	if err != nil {
 		h = host
 	}
 	ip := net.ParseIP(h)
-	ok := ip.To4() != nil // This is always true of IP is IPv4
-
-	// Returns true if input is not IPv4.
-	return !ok
+	return ip.To16() != nil && ip.To4() == nil
 }
 
 // strip api endpoints list with standard ports such as
 // port "80" and "443" before displaying on the startup
 // banner.  Returns a new list of API endpoints.
 func stripStandardPorts(apiEndpoints []string, host string) (newAPIEndpoints []string) {
-	if len(apiEndpoints) == 1 && globalBrowserRedirectURL != nil {
-		if apiEndpoints[0] == globalBrowserRedirectURL.String() {
-			return []string{globalBrowserRedirectURL.String()}
-		}
+	if len(apiEndpoints) == 1 {
+		return apiEndpoints
```

Comment:
in this case we don't remove port 80, 443.

Reply:
That is not removed on purpose since it's custom settings @krisis - we rely on the user to set it right.